org.eclipse.jdt.ls.tests/projects/eclipse/hello/src/com/youcompleteme/Test.java

```diff
@@ -0,0 +1,26 @@
+package com.youcompleteme;
```

Comment:
Sorry but we can't copy this file from YCM, as it is GPL'ed, thus [incompatible with EPL](https://www.eclipse.org/legal/eplfaq.php#GPLCOMPATIBLE).